tests/test_app/config/AltSeeds/NumbersAltSeed.php

```diff
@@ -18,7 +18,6 @@ public function run()
     {
         $data = [
             [
-                'id' => '1',
```

Comment:
Wow really? Do you have documentation for those two cases? I think it might be important to note for users creating seed data.

Note: I've not had that issue on mysql, but mysql is the php of databases so...